monitor/src/main/webapp/resources/monitor/reload.js

```diff
@@ -0,0 +1,23 @@
+window.setInterval(function() {
+	$.ajax({
+		type : "GET",
+		dataType : "json",
+		url : "admin/apps/reload",
```

Comment:
let's make that an absolute url (use the `c:url` tag), otherwise if we for some reason move this page this might break.